h5p-bildetema/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./Footer.module.scss";
+
+export const Footer = (): JSX.Element => {
+  return (
+    <div className={styles.footer}>
+      {/* TODO: Replace with the current language translation */}
+      <h4>Kontaktinformasjon</h4>
```

Comment:
Do we know that `h4` is the correct heading level here? Finding the right h level is a hard problem with no real solution, especially in a component system, but I think maybe `h2` fits even better here. In the same style as the `h4` is.

Reply:
Prefer a `div` or `span` in that case, as `p` has an implicit ARIA `role` that doesn't fit here.